packages/theme-tasks/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@progress/kendo-theme-tasks",
-  "version": "0.5.2",
+  "version": "0.5.3",
```

Comment:
@joneff 
When branched, the version was "0.5.0", due to the new packages, the version automatically went to "0.5.1". During the rebase, I had a conflict as it is "0.5.2" in the "develop" branch. I manually update the version to "0.5.3". Is this okay?

Reply:
yes, it's OK

Reply:
Yes, it's OK, but we'll have to rebase everything one more time